extensions/bitwarden/src/api.ts

```diff
@@ -0,0 +1,38 @@
+import execa from "execa";
+
+export class BitwardenApi {
```

Comment:
```suggestion
export class Bitwarden {
```
`Api` is implied, so it doesn't need to be included.